src/commands/experiments-clear.ts

```diff
@@ -0,0 +1,26 @@
+import { bold } from "colorette";
+
+import { Command } from "../command";
+import * as experiments from "../experiments";
+import { logger } from "../logger";
+
+export const command = new Command("experiments:clear [experiment]")
+  .description("clear preferences for an experiment on this machine")
+  .action((experiment: string) => {
+    if (experiments.isValidExperiment(experiment)) {
+      experiments.setEnabled(experiment, null);
+      experiments.flushToDisk();
+      logger.info(`Cleared preferences for experiment ${bold(experiment)}`);
+      return;
+    }
+
+    logger.error(`Cannot find experiment ${bold(experiment)}`);
+    const potentials = experiments.experimentNameAutocorrect(experiment)!;
+    if (potentials.length === 1) {
+      logger.error(`Did you mean ${potentials[0]}?`);
+    } else if (potentials.length) {
+      logger.error(
+        `Did you mean ${potentials.slice(0, -1).join(",")} or ${potentials[potentials.length - 1]}?`
```

Comment:
Sure, but I also changed the signature of last to never return undefined since that only happened when someone passed a non-array which should never happen due to the signature of the method.

Reply:
I'm an idiot and last can return undefined if the array is non-empty. Will fix